Makefile

```diff
@@ -168,31 +178,49 @@ build-topolvm: build/hypertopolvm build/lvmd
 
 build/hypertopolvm: $(GO_FILES)
 	mkdir -p build
-	go build -o $@ -ldflags "-w -s -X github.com/topolvm/topolvm.Version=$(TOPOLVM_VERSION)" ./pkg/hypertopolvm
+	GOARCH=$(GOARCH) go build -o $@ -ldflags "-w -s -X github.com/topolvm/topolvm.Version=$(TOPOLVM_VERSION)" ./pkg/hypertopolvm
 
 build/lvmd: $(GO_FILES)
 	mkdir -p build
-	CGO_ENABLED=0 go build -o $@ -ldflags "-w -s -X github.com/topolvm/topolvm.Version=$(TOPOLVM_VERSION)" ./pkg/lvmd
+	GOARCH=$(GOARCH) CGO_ENABLED=0 go build -o $@ -ldflags "-w -s -X github.com/topolvm/topolvm.Version=$(TOPOLVM_VERSION)" ./pkg/lvmd
 
 .PHONY: csi-sidecars
 csi-sidecars: ## Build sidecar images.
 	mkdir -p build
-	make -f csi-sidecars.mk OUTPUT_DIR=build
+	make -f csi-sidecars.mk OUTPUT_DIR=build BUILD_PLATFORMS="linux $(GOARCH)"
 
 .PHONY: image
 image: ## Build topolvm images.
-	docker build --no-cache -t $(IMAGE_PREFIX)topolvm:devel --build-arg TOPOLVM_VERSION=$(TOPOLVM_VERSION) .
-	docker build --no-cache -t $(IMAGE_PREFIX)topolvm-with-sidecar:devel --build-arg IMAGE_PREFIX=$(IMAGE_PREFIX) -f Dockerfile.with-sidecar .
+	docker buildx build --no-cache --load -t $(IMAGE_PREFIX)topolvm:devel --build-arg TOPOLVM_VERSION=$(TOPOLVM_VERSION) .
+	docker buildx build --no-cache --load -t $(IMAGE_PREFIX)topolvm-with-sidecar:devel --build-arg TOPOLVM_VERSION=$(TOPOLVM_VERSION) -f Dockerfile.with-sidecar .
 
-.PHONY: tag
-tag: ## Tag topolvm images.
-	docker tag $(IMAGE_PREFIX)topolvm:devel $(IMAGE_PREFIX)topolvm:$(IMAGE_TAG)
-	docker tag $(IMAGE_PREFIX)topolvm-with-sidecar:devel $(IMAGE_PREFIX)topolvm-with-sidecar:$(IMAGE_TAG)
+.PHONY: create-docker-container
+create-docker-container: ## Create docker-container.
+	docker buildx create --use
 
 .PHONY: push
 push: ## Push topolvm images.
-	docker push $(IMAGE_PREFIX)topolvm:$(IMAGE_TAG)
-	docker push $(IMAGE_PREFIX)topolvm-with-sidecar:$(IMAGE_TAG)
+	mkdir -p build
```

Comment:
It is difficult to understand that `make push` does not actually push the image. For example, how about the following?

- `make images`
  - Only build the image
  - Same as `make push` in the current PR
- `make images PUSH=true`
  - Same as `make push PUSH=true` in the current PR
- `make push`
  - Run `make images PUSH=true`

Reply:
> `make images PUSH=true`
Same as `make push PUSH=true` in the current PR

I thought this was a good idea but the task name `image` already exists.
So I changed the task name like below, but what do you think about this task name?

```
$ make multi-platform-images # don't push images
$ make multi-platform-images PUSH=true # build and push images
```

Reply:
Thank you. LGTM.